linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+#check to see if docker is running; if not, start it
+systemctl status docker || systemctl start docker
```

Comment:
sudo

Reply:
added sudo at beginning of both commands in line 4